package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "carbonvotex",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
still 1.0.1?  Since we  changed the structure significantly. I suggest that ii should be bumped up to 2.0.0

Reply:
Agree